source/includes/harvest/_job_openings.md

```diff
@@ -341,11 +344,25 @@ job_id | The ID of the job on which to add new openings.
 
 Parameter | Required | Type | Description
 --------- | ----------- | ----------- | -----------
-openings | yes | Array | This an array of opening IDs, which contain one element.
+openings | yes | Array | This an array of hashes, which can contain an opening ID and custom fields.
 openings.opening_id | yes | Array | This is a string that contains an opening_id. One new opening will be created for each hash element in the array. Opening ID is not required to have a string value and may be null.  Greenhouse has an internal limit of 100 open openings. If you attempt to create more than 100 openings in a single request, or if this request would create more than 100 open openings, the request will fail.
+openings.custom_fields[] | No | Array | Array of hashes containing new custom field values. Passing an empty array does nothing. If you have any required custom fields configured for openings, they must be supplied for each new opening, or the request will fail.
 
 **Note**: Adding new openings may re-trigger approvals. For approvals to start recruiting, this will reset approvals only if the job is in draft mode. If the job is open for hiring, these approvals will not reset. For official job approvals, this will reset approvals only if the job is open.
 
+### Custom Field Parameters
+
+The custom field parameter structure is different in the POST method than in GET methods and responses. Certain types of custom fields require different elements to be included. See below for the description of each item in a custom field element and what is required depending on the type.
+
+Parameter | Required for | Description
+---------- | -------------- | ----------------
+id | all | The custom field id for this particular custom field.  One of this or name_key is required.
```

Comment:
I suggest rephrasing this to match how we talk about the "this or that" requirement for `name_key`:

```
This or id is required for all custom field elements.
```

I think saying "all custom field elements" could be a bit confusing; saying something like "regardless of custom field type" might be clearer.

So something like:

```
`id` or `name_key` is required, regardless of custom field type.
```